mmv1/third_party/terraform/services/logging/resource_logging_project_bucket_config.go

```diff
@@ -190,6 +195,9 @@ func resourceLoggingProjectBucketConfigAcquireOrCreate(parentType string, iDFunc
 				log.Printf("[DEGUG] Loggin Bucket not exist %s", id)
 				// we need to pass the id in here because we don't want to set it in state
 				// until we know there won't be any errors on create
+				if d.Get("enable_analytics") {
```

Comment:
Why is enable_analytics the trigger for the async creation mode?

Reply:
this is the implementation details. We do use `enable_analytics` to decide whether we should use async api or not.

and we already had customers using non-async API then encountered time-out issue because the operation is not finished in 2 mins.

We don't recommend calling async APIs for non-analytics-enabled buckets. Is it possible that we can do the switch here?

Thanks!

Reply:
Why is it not recommended to use the async APIs for non-analytics buckets?

Reply:
sorry after asking around, the final answer is there is no downside for using `CreateBucketAsync` everywhere. `CreateBucketAsync` is strictly better than `CreateBucket`(sync). The only reason we kept the latter is for backwards compatibility reasons.

I'll change the pr accordingly.